hadoop-ozone/ozonefs/src/main/java/org/apache/hadoop/fs/ozone/FilteredClassLoader.java

```diff
@@ -57,6 +57,12 @@ public FilteredClassLoader(URL[] urls, ClassLoader parent) {
     delegatedClasses.add("org.apache.hadoop.fs.ozone.OzoneFSStorageStatistics");
     delegatedClasses.add("org.apache.hadoop.fs.ozone.Statistic");
     delegatedClasses.add("org.apache.hadoop.fs.Seekable");
+    String[] dynamicDelegatedClasses =
+        System.getProperty("HADOOP_OZONE_DELEGATED_CLASSES").split(";");
+    for (String delegatedClass : dynamicDelegatedClasses) {
+      delegatedClasses.add(delegatedClass);
+    }
```

Comment:
You can replace the code L60-64 with 
delegatedClasses.addAll(
        getTrimmedStringCollection(System.getenv("HADOOP_OZONE_DELEGATED_CLASSES")));

Reply:
getTrimmedStringCollection is member function of class Configuration. We can't directly use here. I can add trim() if we care about the space. Is that ok?

Reply:
I see.